src/mots/cli.py

```diff
@@ -396,11 +396,11 @@ def create_parser(subcommand=None):
 
     parsers["search"].add_argument("match", nargs=1, help="a search string")
     parsers["check-for-updates"].add_argument(
-        "--include-dev-releases",
+        "--include-pre-releases",
         "-d",
```

Comment:
Does `-d` still make sense for the short flag?

Reply:
Good point, I will change it.